release/nightly_tests/dataset/iter_tensor_batches_benchmark.py

```diff
@@ -103,7 +105,7 @@ def run_iter_tensor_batches_benchmark(benchmark: Benchmark, data_size_gb: int):
             batch_size=batch_size,
         )
 
-    prefetch_batches = [1, 10]
+    prefetch_batches = [1, 4]
```

Comment:
any reason why are changing this value now?

Reply:
Should’ve been changed beforehand

Reply:
sorry why we want to decrease the value from 10 to 4 here?

Reply:
When I had tried this benchmark manually, I didn’t really see much improvement with 10.

I was meaning to change this to 4, but it had just slipped.

But we can keep both, there’s no harm.